ssl/test/bssl_shim.cc

```diff
@@ -305,6 +305,9 @@ static bool CheckListContains(const char *type,
   std::vector<const char *> list(list_func(nullptr, 0));
   list_func(list.data(), list.size());
   for (const char *expected : list) {
+    if (str == nullptr) {
```

Comment:
should we also check `expected`?

Reply:
The list can't contain `NULL` because `list_func` callback just returns some static list of names. basically.

If you insist, I could.